sections/collections__products-custom.liquid

```diff
@@ -1,29 +1,12 @@
 
-<!-- <div class="product-grid">
-    <ul class="display-grid">
-        {% for product in collection.products %}
-            <a href="{{ product.url }}">
-                <li class="product-grid__list-item">
-                    <img class="featured-image" src="{{ product.featured_image | img_url: 'medium' }}">
-                    {% for image in product.images %}
-                        {% if forloop.index0 == 1 %}
-                            <img class="hovered-image" src="{{ image | img_url: 'medium' }}">
-                        {% endif %}
-                    {% endfor %}
-                    <h4 class="product-grid__title">{{ product.title }}</h4>
-                    <h5 class="product-grid__vendor">{{ product.vendor }}</h5>
-                    <h6 class="product-grid__price">{{ product.price | money }}</h6>
-                </li>
-            </a>
-        {% endfor %}
-    </ul>
-</div> -->
 <div class="product-grid">
     <ul class="display-grid">
         {% for product in collection.products %}
             <a href="{{ product.url }}">
                 <li class="product-grid__list-item">
-                    <img class="featured-image" src="{{ product.featured_image | img_url: 'medium' }}" onmouseover="this.src='{% for image in product.images %}{% if forloop.index0 == 1 %}{{ image | img_url: 'medium' }}{% endif %}{% endfor %}'" onmouseout="this.src='{{ product.featured_image | img_url: 'medium' }}'"/>
+                    <img class="featured-image" src="{{ product.featured_image | img_url: 'medium' }}">
+                    <input type="hidden" id="initial-image" value="{{ product.featured_image | img_url: 'medium' }}">
+                    <input type="hidden" id="hovered-image" value="{% for image in product.images %}{% if forloop.index0 == 1 %}{{ image | img_url: 'medium' }}{% endif %}{% endfor %}">
```

Comment:
this def works but i'd probably refactor it into 
`{% for image in product.images %} ....
{% if forloop.index < 2 %}
 <input type="hidden"..... />
<input type.... />
...
{% endif %}
{% endfor %}`